src/lib/Helpers.coffee

```diff
@@ -100,12 +116,16 @@ exports.WirePattern = (component, config, proc) ->
   config.forwardGroups = false unless 'forwardGroups' of config
```

Comment:
That will break some things. I'd like to keep backwards compatibility layer as backwards compatible as possible.